<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OcmeDockWidget</class>
 <widget class="QDockWidget" name="OcmeDockWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>491</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>800</height>
   </size>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Attributes in OCM</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ocmeAttrListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="meshAttrListWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Attributes in trimesh</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>273</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="loadOcmPushButton">
       <property name="toolTip">
        <string>set the current out-of-core database</string>
       </property>
       <property name="text">
        <string>load OCM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createOcmPushButton">
       <property name="text">
        <string>new OCM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeOcmPushButton">
       <property name="text">
        <string>close OCM</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="fillAttrMeshPushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tri2ocmPushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ocm2triPushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshImpostorsPushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh impostors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleExtrPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>93</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>toggle extr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleShowTouchedPushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>toggle show touched</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>160</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="commitPushButton">
       <property name="text">
        <string>Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dropSelectionPushButton">
       <property name="text">
        <string>Drop ROI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editPushButton">
       <property name="text">
        <string>Edit ROI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="markEditablePushButton">
       <property name="text">
        <string>Select Editable</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>160</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>Add Current Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addFromDiskPushButton">
       <property name="text">
        <string>Add From disk</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="editAllPushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>edit all (globall)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyPushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>verify (gloabl)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="onlyImpostorsCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Only impostors</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="videoramSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>video RAM (MB)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
